.github/workflows/build.yaml

```diff
@@ -603,8 +603,9 @@ jobs:
 
     - name: Scan
       run: |
-        go install golang.org/x/vuln/cmd/govulncheck@latest
-        find bin/linux_amd64 -maxdepth 1 -type f -exec govulncheck -mode=binary -json {} \; | tee vulns.json
+        GOVULNCHECK_BIN=$PWD/.gotools/bin/govulncheck
+        make "$GOVULNCHECK_BIN"
```

Comment:
Hardcoding this here is not nice, but currently necessary.

Definitely not in scope of this PR, but would you agree it would be nice to have:
- a simple `.PHONY` target name to run, perhaps just `GOVULNCHECK_BIN` in this case?
- another target what would just echo the path to the binary?

Reply:
Fixed

Reply:
Oh, wow, we already do!